File: app/src/main/java/com/certified/notes/model/Todo.java
package com.certified.notes.model;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "todo_table")
public class Todo {

    @PrimaryKey(autoGenerate = true)
    private int id;

    private final String todo;

    private final boolean done;

    public Todo(String todo, boolean done) {
        this.todo = todo;
        this.done = done;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTodo() {
        return todo;
    }

    public boolean isDone() {
        return done;
    }
}

File: app/src/main/java/com/certified/notes/model/Todo.kt
package com.certified.notes.model

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "todo_table")
data class Todo(
    val todo: String,
    val isDone: Boolean
) {
    @PrimaryKey(autoGenerate = true)
    var id: Int = 0
}